test/test_other.py

```diff
@@ -9406,14 +9406,6 @@ def test_exceptions_stack_trace_and_message(self):
       'at (src.wasm.)?foo',
       'at (src.wasm.)?main']
 
-    if '-fwasm-exceptions' in self.emcc_args:
-      # FIXME Node v18.13 (LTS as of Jan 2023) has not yet implemented the new
```

Comment:
So we don't run node 18 anymore on these tests?

Reply:
I believe that's the case yes. It's certainly the case for Chromium CI. The github checks will be the final arbiter here :)